README.md

```diff
@@ -218,9 +218,11 @@ This method has 2 overloads.
 
 ### EquiZip
 
-Returns a projection of tuples, where each tuple contains the N-th
-element from each of the argument sequences. An exception is thrown
-if the input sequences are of different lengths.
+Returns a sequence of projections, each projection is build from K elements.
```

Comment:
I'm afraid that these new descriptions read much more complicated by introducing K into the picture in addition to N already. Do these really need revision as part of this _refactoring_?

Reply:
Are you ok with "multiple" instead of K ?
I'm not ok with 'tuples' in the original description, it's misleading since no tuple object are created, and the result selector doesn't accept a tuple as input argument.

Reply:
> Are you ok with "multiple" instead of K ?

Definitely better than K.